etc/error_code.toml

```diff
@@ -1,550 +1,550 @@
-error = 'KV-Storage-EmptyRequest'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Closed]
-error = 'KV-Storage-Closed'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Io]
-error = 'KV-Storage-Io'
-description = ''
-workaround = ''
-
-[error.KV-Storage-SchedTooBusy]
-error = 'KV-Storage-SchedTooBusy'
-description = ''
-workaround = ''
-
-[error.KV-Storage-GcWorkerTooBusy]
-error = 'KV-Storage-GcWorkerTooBusy'
-description = ''
-workaround = ''
-
-[error.KV-Storage-KeyTooLarge]
-error = 'KV-Storage-KeyTooLarge'
-description = ''
-workaround = ''
-
-[error.KV-Storage-InvalidCF]
-error = 'KV-Storage-InvalidCF'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Protobuf]
-error = 'KV-Storage-Protobuf'
-description = ''
-workaround = ''
-
-[error.KV-Storage-INVALIDTXNTSO]
-error = 'KV-Storage-INVALIDTXNTSO'
-description = ''
-workaround = ''
-
-[error.KV-Storage-InvalidReqRange]
-error = 'KV-Storage-InvalidReqRange'
-description = ''
-workaround = ''
-
-[error.KV-Storage-BadFormatLock]
-error = 'KV-Storage-BadFormatLock'
-description = ''
-workaround = ''
-
-[error.KV-Storage-BadFormatWrite]
-error = 'KV-Storage-BadFormatWrite'
-description = ''
-workaround = ''
-
-[error.KV-Storage-KeyIsLocked]
-error = 'KV-Storage-KeyIsLocked'
-description = ''
-workaround = ''
-
-[error.KV-Storage-MaxTimestampNotSynced]
-error = 'KV-Storage-MaxTimestampNotSynced'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Committed]
-error = 'KV-Storage-Committed'
-description = ''
-workaround = ''
-
-[error.KV-Storage-PessimisticLockRolledBack]
-error = 'KV-Storage-PessimisticLockRolledBack'
-description = ''
-workaround = ''
-
-[error.KV-Storage-TxnLockNotFound]
-error = 'KV-Storage-TxnLockNotFound'
-description = ''
-workaround = ''
-
-[error.KV-Storage-TxnNotFound]
-error = 'KV-Storage-TxnNotFound'
-description = ''
-workaround = ''
-
-[error.KV-Storage-LockTypeNotMatch]
-error = 'KV-Storage-LockTypeNotMatch'
-description = ''
-workaround = ''
-
-[error.KV-Storage-WriteConflict]
-error = 'KV-Storage-WriteConflict'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Deadlock]
-error = 'KV-Storage-Deadlock'
-description = ''
-workaround = ''
-
-[error.KV-Storage-AlreadyExist]
-error = 'KV-Storage-AlreadyExist'
-description = ''
-workaround = ''
-
-[error.KV-Storage-DefaultNotFound]
-error = 'KV-Storage-DefaultNotFound'
-description = ''
-workaround = ''
-
-[error.KV-Storage-CommitTsExpired]
-error = 'KV-Storage-CommitTsExpired'
-description = ''
-workaround = ''
-
-[error.KV-Storage-KeyVersion]
-error = 'KV-Storage-KeyVersion'
-description = ''
-workaround = ''
-
-[error.KV-Storage-PessimisticLockNotFound]
-error = 'KV-Storage-PessimisticLockNotFound'
-description = ''
-workaround = ''
-
-[error.KV-Storage-CommitTsTooLarge]
-error = 'KV-Storage-CommitTsTooLarge'
-description = ''
-workaround = ''
-
-[error.KV-Storage-Unknown]
-error = 'KV-Storage-Unknown'
-description = ''
-workaround = ''
+["KV:Codec:Io"]
+error = '''
+KV:Codec:Io
+'''
```

Comment:
It is intended that there is a trailing newline?

Reply:
It's OK, we can trim space when the tiup errdoc component loads them.